packages/custom_lint_core/test/configs_test.dart

```diff
@@ -338,6 +359,68 @@ foo:
       });
     });
 
+    test('Parses error severities from configs', () {
+      final analysisOptions = createAnalysisOptions('''
+custom_lint:
+  errors:
+    rule_name_1: error
+    rule_name_2: warning
+    rule_name_3: info
+    rule_name_4: none
```

Comment:
Afaik analyzer uses `ignore` not `none`:

```
errors:
  unused_parameter: ignore
```

We should match

Reply:
Yeah, I've missed that fact! I will refactor it :)